src/main.cc

```diff
@@ -59,7 +63,16 @@ int main(int argc, char *argv[]) {
             use_captured_sym_map = map_type == 2;
             break;
         case 'v':
-            verbosity = atoi(optarg);
+            verb_trim = strtok(optarg, " ");
```

Comment:
This code is a bit hard to parse -- any chance we could factor out into a helper function and then test that function?

Reply:
Sure, I will put it into a separate function. By testing it do you mean making unit tests for the command line only program?

Reply:
Ok, I used a handy C++ unit test library named Catch (https://github.com/catchorg/Catch2) to make some unit tests for this function. I also set it up so that we can continue to add unit tests for the whole command line program in the future. To run the tests, do 'make tests' from the src folder, followed by './tests'.